numpy/core/numerictypes.py

```diff
@@ -373,14 +375,44 @@ def issubdtype(arg1, arg2):
     See Also
     --------
     issubsctype, issubclass_
-    numpy.core.numerictypes : Overview of numpy type hierarchy.
+    arrays.scalars : Overview of numpy type hierarchy.
```

Comment:
This link still doesn't work - any ideas?

Reply:
Ping @rossbar, who may know.

Reply:
This links to code objects. `numpy.core.numerictypes` is a thing, `arrays.scalars` is not. You'll need `:ref:` I think.

Reply:
`numpy.core.numerictypes` is not a thing that exists in the generated documentation.

Reply:
Hopefully fixed, let's see...

Reply:
Yep, fixed